apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -422,16 +426,25 @@ const RemixLine: React.FC<RemixLineProps> = ({
                   <ul className="">
                     {Object.entries(toolTiplabels).map(([key, name]) => {
                       const value = data[key];
+                      console.log("key", key, "value", value, "data", data[key]);
                       if (key === "DELTA" && !deltaView) return null;
                       if (typeof value !== "number") return null;
                       if (deltaView && key === "GENERATION" && data["GENERATION_UPDATED"] >= 0)
                         return null;
                       if (key.includes("4HR") && (!show4hView || !visibleLines.includes(key)))
                         return null;
-
-                      const textClass = ["FORECAST", "PAST_FORECAST"].includes(name)
+                      if (key.includes("PROBABILISTIC") && Math.round(value * 100) < 0) return null;
+                      const textClass = ["FORECAST", "PAST_FORECAST"].includes(key)
                         ? "font-semibold"
+                        : ["PROBABILISTIC_UPPER_BOUND", "PROBABILISTIC_LOWER_BOUND"].includes(key)
+                        ? "text-xs"
```

Comment:
This is fine here, but in general my personal pref is against nested ternary statements – they are tricky to read esp when it's not your code, and can usually be written (almost) as concisely with a `let textClass = "default-class"` statement and then some ifs to change where applicable, if that makes sense?